src/studio/nuxeo-online-services-portal.md

```diff
@@ -138,4 +138,5 @@ The documentation tab provides a guide on how to create JIRA tickets to benefit
 
 **RELATED TOPICS**
 
-[Nuxeo Support issue Tracker: JIRA]({{page page='support-jira'}})
+- [Nuxeo Support issue Tracker: JIRA]({{page page='support-jira'}})
+- [Nuxeo Studio Early Access Program]({{page version='' space='studio' page='early-access-program'}})
```

Comment:
It would be better to put a link to the early access page directly on the sentence saying "and information about the early access program at a glance." in the dashboard section.

Reply:
Done